global.d.ts

```diff
@@ -1,7 +1,7 @@
 /// <reference types="@hashicorp/platform-types" />
 
 declare module 'swingset/page' {
-  const createPage: () => $TSFixMe
+  const createPage: (opts: $TSFixMe) => $TSFixMe
```

Comment:
Required so we can pass custom `{ components }` for use in Swingset pages. Specifically, I'm showing `@hashicorp/react-search` on the [`header-search-input`](https://dev-portal-git-zsadd-dev-tooltip-hashicorp.vercel.app/swingset/components/headersearchinput) page for functionality demo purposes (and as a starting point for upcoming implementation work, which will definitely leverage the package, including `<SearchProvider />`).